src/utils/rfqm_health_check.ts

```diff
@@ -0,0 +1,50 @@
+export enum HealthCheckStatus {
+    Operational = 'operational',
+    Unknown = 'unknown',
```

Comment:
still unclear to me how to use the 'unknown',  a component that can't report its status I would assume is down

Reply:
I'll cut it if we end up not needing it (see the discussion in the design about when this would be used).